pkg/scraper/mapped.go

```diff
@@ -276,11 +278,33 @@ func (p *postProcessMap) Apply(value string, q mappedQuery) string {
 	return value
 }
 
+type postProcessToMetric bool
+
+func (p *postProcessToMetric) Apply(value string, q mappedQuery) string {
+	const foot_in_cm = 30.48
+	const inch_in_cm = 2.54
+
+	reg := regexp.MustCompile("[0-9]+")
+	filtered := reg.FindAllString(value, -1)
+
+	var parsed []float64
+	for _, v := range filtered {
+		k, _ := strconv.ParseFloat(v, 64)
+		parsed = append(parsed, k)
+	}
+
+	var centimeters = parsed[0]*foot_in_cm + parsed[1]*inch_in_cm
+
+	// Return rounded integer string
+	return strconv.Itoa(int(math.Round(centimeters)))
+}
+
 type mappedPostProcessAction struct {
 	ParseDate  string                   `yaml:"parseDate"`
 	Replace    mappedRegexConfigs       `yaml:"replace"`
 	SubScraper *mappedScraperAttrConfig `yaml:"subScraper"`
 	Map        map[string]string        `yaml:"map"`
+	ToMetric   bool                     `yaml:"toMetric"`
```

Comment:
I think `toMetric` is too generic for this purpose. `feetToCm` would probably be more suitable.

Reply:
This was my first idea as well, but then I noticed it's feet and inches to cm :D Anyway, I will take your suggestion as it's more intuitive.